doc/cephfs/fuse.rst

```diff
@@ -2,7 +2,7 @@
 Mount CephFS using FUSE
 =======================
 
-Before mounting a Ceph File System in User Space (FUSE), ensure that the client
+Before mounting a Ceph Filesystem in User Space (FUSE), ensure that the client
```

Comment:
Didn't @batrick just do a PR that did s/Filesystem/File System/? I think you probably don't want to revert that.

Reply:
Shouldn't the abbreviation expansion for FUSE be unchanged? : http://man7.org/linux/man-pages/man8/mount.fuse.8.html

Reply:
I'm fine with reverting this particular change. Put it in quotes: `"Filesystem in USErspace"`